EIPS/eip-6682.md

```diff
@@ -0,0 +1,161 @@
+---
+eip: 6682
+title: NFT Flashloans
+description: Minimal interface for ERC-721 NFT flashloans
+author: out.eth (@outdoteth)
+discussions-to: https://ethereum-magicians.org/t/eip-6682-nft-flashloans/13294
+status: Draft
+type: Standards Track
+category: ERC
+created: 2023-03-12
+requires: 721, 3156
```

Comment:
```suggestion
requires: 20, 721, 3156
```

The fees are paid in ERC-20 tokens, so it's a hard requirement.

Reply:
updated